nurseconnect/templates/core/section_page.html

```diff
@@ -7,62 +6,90 @@
 {% block body_class %}template-{{ self.get_verbose_name|slugify }}{% endblock %}
 
 {% block content %}
-    <div class="articles nav {{self.get_effective_extra_style_hints}}">
-      <h2 class="list-header">
+    <div class="articles nav {{ self.get_effective_extra_style_hints }}">
         {% if self.image %}
-          <a href="{% pageurl self %}">{% image self.image width-60 %}</a>
+            <a href="{% pageurl self %}">{% image self.image width-60 %}</a>
```

Comment:
I could not find an example on the FED styleguide for image styling on the sections list, also i'm not sure if thats going to be a feature since there hasn't been confirmation on it yet. So I will just comment it out for now.